public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Cromwell Pipeline",
+  "name": "Cromwell Pipeline",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "256x256",
```

Comment:
I recommend looking at [this](https://habr.com/ru/post/522844/) and running away in horror ;)

Reply:
Ahah, yes that's interesting :D
I've used [this one](https://realfavicongenerator.net/) for favicon generation
Hope we might trust this resource